.snippets/code/develop/interoperability/xcm-runtime-apis/xcm-example.js

```diff
@@ -0,0 +1,84 @@
+import { createClient } from 'polkadot-api';
+import { getWsProvider } from 'polkadot-api/ws-provider/web';
+import { withPolkadotSdkCompat } from 'polkadot-api/polkadot-sdk-compat';
+import {
+  XcmVersionedXcm,
+  paseoAssetHub,
+  XcmVersionedLocation,
+  XcmV3Junction,
+  XcmV3Junctions,
+  XcmV3WeightLimit,
+  XcmV3MultiassetFungibility,
+  XcmV3MultiassetAssetId,
+  XcmV3Instruction,
+  XcmV3MultiassetMultiAssetFilter,
+  XcmV3MultiassetWildMultiAsset,
+} from '@polkadot-api/descriptors';
+import { Binary } from 'polkadot-api';
+import { ss58Decode } from '@polkadot-labs/hdkd-helpers';
+
+// Connect to Paseo Asset Hub
+const client = createClient(
+  withPolkadotSdkCompat(getWsProvider('wss://asset-hub-paseo-rpc.dwellir.com')),
+);
+
+const paseoAssetHubApi = client.getTypedApi(paseoAssetHub);
+
+const userAddress = 'INSERT_USER_ADDRESS';
+const userPublicKey = ss58Decode(userAddress)[0];
+const idBenef = Binary.fromBytes(userPublicKey);
+
+// Define the origin
+const origin = XcmVersionedLocation.V3({
+  parents: 1,
+  interior: XcmV3Junctions.Here(),
+});
+
+// Define a xcm message comming from the Paseo relay chain to Asset Hub to Teleport some tokens
+const xcm = XcmVersionedXcm.V3([
```

Comment:
Can we continue the previous example and instead of creating a message from scratch we use the xcm we got from the last part?

Reply:
The idea is that each example is self-contained. Another issue is that Pop Network does not implement the xcm runtime APIs, so we cannot execute the queries there.